lms/djangoapps/bookmarks/api.py

```diff
@@ -0,0 +1,35 @@
+"""
+This File contains bookmark api method(s).
+"""
+from opaque_keys.edx.keys import UsageKey
+
+from bookmarks.serializers import BookmarkSerializer
+from bookmarks.models import Bookmark
+
+from xmodule.modulestore.django import modulestore
+
+
+def get_bookmark(requested_user, usage_id, fields_to_add=None, serialized=True):
+    """
+    Returns bookmark object or JSON response.
+
+    Args:
+        requesting_user (User): The user requesting the bookmark.
+        fields_to_add (list): List of fields to return for a bookmark.
+        usage_id (str): The usage id of an Xblock.
```

Comment:
This function should be passed a `usage_key` and not a `usage_id`. The caller should do the conversion.